data/data/libvirt/cluster/output.tf

```diff
@@ -0,0 +1,19 @@
+output "pool" {
+  value = libvirt_pool.storage_pool.name
+}
+
+output "base_volume_id" {
+  value = libvirt_volume.coreos_base.id
+}
+
+output "network_id" {
+  value = libvirt_network.net.id
+}
+
+output "bootstrap_ip" {
```

Comment:
This should probably be output in the bootstrap stage.

Reply:
Good idea, and done. Ironically... these IPs are set in the installer, passed into Terraform, and then output back to the installer.